test/edu-benefits/components/IntroductionPage.unit.spec.jsx

```diff
@@ -1,34 +0,0 @@
-import React from 'react';
-import { findDOMNode } from 'react-dom';
-import { expect } from 'chai';
-import sinon from 'sinon';
-import ReactTestUtils from 'react-dom/test-utils';
-import { IntroductionPage } from '../../../src/js/edu-benefits/1995/components/IntroductionPage';
-
-describe('Edu 1995 <IntroductionPage>', () => {
```

Comment:
Only the 1995 had this test, which was the only test in it's directory. The e2e tests seem to supersede this, please let me know if it makes sense to revert this removal.

Reply:
I think that's fine, that test is outdated. I'm going to create a tech debt ticket to write new unit tests for the intro pages, though. We should have some.

Reply:
Thanks for the recommendation, I've created the issue [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4864).